umbrella/src/lib.rs

```diff
@@ -1164,6 +1164,9 @@ pub use sc_telemetry;
 #[cfg(feature = "sc-tracing")]
 pub use sc_tracing;
 
+#[cfg(feature = "with-tracing")]
+pub use sc_tracing as sc_tracing_with;
```

Comment:
Perhaps you are experimenting with something yet, but I don't get what is going on here.